src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+@import "../utils/variables";
+
+.card {
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $background-color-white;
+  margin-right: 48px;
+
+  &__img {
+    display: flex;
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+    top: 33px;
+  }
+
+  &__desciption-text {
+    display: flex;
+    justify-content: center;
+    margin: 0 16px;
+    font-weight: 500;
+    color: $text-color;
+    font-size: 12px;
+    line-height: 18px;
+    left: calc(50% - 166px / 2);
+    bottom: 165px;
```

Comment:
redundant - delete it

![image](https://github.com/mate-academy/layout_catalog/assets/86923089/9f1fe20a-e0b7-4a14-8492-26b5eb445b1b)